src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -184,6 +185,7 @@ export class MongoStorageAdapter implements StorageAdapter {
         // Starting mongoDB 3.0, the MongoClient.connect don't return a DB anymore but a client
         // Fortunately, we can get back the options and use them to select the proper DB.
         // https://github.com/mongodb/node-mongodb-native/blob/2c35d76f08574225b8db02d7bef687123e6bb018/lib/mongo_client.js#L885
+        client.appendMetadata({ name: 'parse_server_storage', version: version });
```

Comment:
@coderabbitai Would the metadata be sent to the MongoDB cluster on every request, or only once when initializing the MongoDB driver - cluster connection?